scripts/aws/cfn/storage.yml

```diff
@@ -85,9 +106,11 @@ Outputs:
   DataIndexerLambdaName:
     Value: SenderDashboardDataIndexer
   # Bucket exports
-  IndexesBucketArn:
-    Value: !GetAtt IndexesBucket.Arn
-  IndexesBucketName:
-    Value: !Ref IndexesBucket
-  IndexesBucketRegion:
-    Value: !Ref AWS::Region
\ No newline at end of file
+  BffBucketArn:
```

Comment:
Aggiornare IndexesBucketArn, IndexesBucketName e IndexesBucketRegion con in nuovi nomi anche dentro microservice.yml